consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/payload/QbftPayload.java

```diff
@@ -29,4 +31,14 @@ protected void writeConsensusRound(final RLPOutput out) {
   protected static ConsensusRoundIdentifier readConsensusRound(final RLPInput in) {
     return new ConsensusRoundIdentifier(in.readLongScalar(), in.readIntScalar());
   }
+
+  @Override
+  public Hash hashForSignature() {
```

Comment:
Can you add a test for this hashing

Reply:
without just reproducing the literal Rlp-ing, not sure how.